lumeer-core/src/main/java/io/lumeer/core/facade/ViewFacade.java

```diff
@@ -101,6 +101,13 @@ public View getViewByCode(final String code) {
       final View view = viewDao.getViewByCode(code);
       permissionsChecker.checkRole(view, Role.READ);
 
+      if (view.getAuthorId() == null || "".equals(view.getAuthorId())) {
+         if (permissionsChecker.hasRole(view, Role.MANAGE)) {
+            view.setAuthorId(authenticatedUser.getCurrentUserId());
+            viewDao.updateView(view.getId(), view);
+         }
+      }
```

Comment:
This method (getViewByCode) is never called in frontend so this code doesn't heal anything.

Reply:
It is called in ViewService in getView(). This is not used in frontend?

Reply:
No

Reply:
So what method is used to actually read the complete view?

Reply:
And how about now?

Reply:
It's better now :)